shared/style/headers.css

```diff
@@ -285,8 +285,8 @@ section[role="region"].skin-dark > header:first-child button:focus {
 
 section[role="region"].skin-dark header h2,
 .skin-dark > section[role="region"] header h2 {
-  background-color: #2c353b;
-  color: #fff;
+  background-color: #e7e7e7;/* change #2c353b to #e7e7e7*/
```

Comment:
I'm not sure, that such comments are required here, all history anyway exists in GIT. It could, probably, make sence in case if in the comment will be mentioned not only the change itself, but also reason of that change; but even in this case using of comments for storeing of history - isn't an optimal way.